Source/ARTPushAdmin+Private.h

```diff
@@ -9,9 +7,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ARTPushAdminInternal : NSObject <ARTPushAdminProtocol>
 
-@property (nonatomic, readonly) ARTPushDeviceRegistrationsInternal *deviceRegistrations;
```

Comment:
Moving these properties is a change to the public API, and also [these properties are described in the features spec](https://ably.com/documentation/client-lib-development-guide/features#RSH1); we’d need to update the features spec before making this change and need to make sure we do a new major release of the library afterwards.